src/evo/deterministicmns.h

```diff
@@ -79,29 +79,8 @@ class CDeterministicMN
         READWRITE(VARINT(internalId));
         READWRITE(collateralOutpoint);
         READWRITE(nOperatorReward);
-        // We need to read CDeterministicMNState using the old format only when called with MN_OLD_FORMAT or MN_TYPE_FORMAT on Unserialize()
```

Comment:
Maybe throw some error here if `format_version != MN_TYPE_FORMAT`?
+if (format_version != MN_TYPE_FORMAT) throw some error

Reply:
Resolved in latest push!